.eslintrc

```diff
@@ -0,0 +1,13 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es2021": true,
+        "node": true
+    },
+    "parserOptions": {
+        "ecmaVersion": 12
+    },
+    "rules": {
+    }
+}
```

Comment:
Can we actually inherit the `.eslintrc` from NodeBB core just to be consistent? :smile: